repos/spack_repo/builtin/packages/curl/package.py

```diff
@@ -374,3 +385,55 @@ def install(self, pkg, spec, prefix):
             # safeguard against future curl releases that do this for us
             if os.path.exists(libcurl_a) and not os.path.exists(libcurl):
                 symlink(libcurl_a, libcurl)
+
+
+class CMakeBuilder(CMakeBuilder):
+    def cmake_args(self):
+        args = [
+            self.define("BUILD_TESTING", False),
+            self.define("CURL_USE_LIBPSL", False),
+            # Curl's CMake will turn this off if not building static libcurl
+            self.define("BUILD_STATIC_CURL", True),
+            # enables install from cmake
+            self.define("CURL_DISABLE_INSTALL", False),
+            self.define("BUILD_MISC_DOCS", False),
+            self.define("BUILD_LIBCURL_DOCS", False),
+            self.define("BUILD_EXAMPLES", False),
+            self.define("CURL_BROTLI", False),
+            self.define("CURL_USE_GSASL", False),
+            self.define("CURL_ZSTD", False),
+            self.define("ENABLE_CURL_MANUAL", False),
+            self.define_from_variant("CURL_USE_LIBSSH2", "libssh2"),
+            self.define_from_variant("CURL_USE_LIBSSH", "libssh"),
+            self.define_from_variant("CURL_USE_OPENLDAP", "ldap"),
+            self.define_from_variant("CURL_DISABLE_LDAP", "ldap"),
+            self.define_from_variant("USE_NGHTTP2", "nghttp2"),
+            self.define_from_variant("CURL_USE_GSSAPI", "gssapi"),
+            self.define_from_variant("USE_LIBRTMP", "librtmp"),
+            self.define_from_variant("USE_LIBIDN2", "libidn2"),
+        ]
+
+        if self.spec.satisfies("tls=sspi"):
+            self.define("CURL_WINDOWS_SSPI", True)
```

Comment:
```suggestion
        self.define("CURL_WINDOWS_SSPI", self.spec.satisfies("tls=sspi"))
```
as part of  `args = [...]`.

Right now you're producing an unused string.

Reply:
doh!